src/libraries/System.Net.Http/src/System/Net/Http/SocketsHttpHandler/HttpContentWriteStream.cs

```diff
@@ -24,10 +24,10 @@ private abstract class HttpContentWriteStream : HttpContentStream
 
             public sealed override Task FlushAsync(CancellationToken ignored)
             {
-                HttpConnection connection = _connection;
+                HttpConnection? connection = _connection;
                 return connection != null ?
                     connection.FlushAsync().AsTask() :
-                    default;
```

Comment:
Base not returning nullable

Reply:
Hmm, could this not be changed to `Task.CompletedTask` to avoid surprises?

Reply:
I assume `connection == null` is not expected error nous case so its returning default so that it throws on calling side, so not sure if I should make such change CC @dotnet/ncl @stephentoub